apisix/plugins/fault-injection.lua

```diff
@@ -60,7 +60,7 @@ local function sample_hit(percentage)
         return true
     end
 
-    return random(0, 100) <= percentage
+    return random(1, 100) <= percentage
```

Comment:
do we need to update the `doc`?

Reply:
Current behavior is the correct one according to the doc. Previously, fault-injection will happen even percentage is zero.

Reply:
got it